tests/cyclonedx.tests/MergeTests.cs

```diff
@@ -63,11 +63,11 @@ public class MergeTests
                 };
                 foreach (var inputFilename in inputFilenames)
                 {
-                    options.InputFiles.Add(Path.Combine("Resources", "Merge", inputFilename));
+                    options.InputFiles.Insert(0, Path.Combine("Resources", "Merge", inputFilename));

```

Comment:
Need to look more into this. Something flips the order of the BOMs at some point and this was a bad attempt to correct it

Reply:
Forgot to add the break to the loop I created, so it keep updating the data.